### PR TITLE
STEM: Notify applicant when contacting SCO #17363

### PR DESCRIPTION
**Pull request by zurbergram**
## Description of change

Send an email to applicant when SCOs are notified because CCing is not available.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#17363

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -322,6 +322,15 @@
     t.index ["job_id"], name: "index_form526_job_statuses_on_job_id", unique: true
   end
 
+  create_table "form526_opt_ins", id: :serial, force: :cascade do |t|
```
Comment (zurbergram) [inline db/schema.rb:325]: This change appears every time a `make rebuild` is done

File: lib/config_helper.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ module ConfigHelper
 
   def setup_action_mailer(config)
     config.action_mailer.preview_path = Rails.root.join('spec', 'mailers', 'previews')
+    config.action_mailer.show_previews = Rails.env.development? || FeatureFlipper.staging_email?
```
Comment (zurbergram) [inline lib/config_helper.rb:10]: this should make both http://localhost:3000/rails/mailers and https://staging-api.va.gov/rails/mailers/ available to use.  This should not enable this in production

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: lib/config_helper.rb (hunk 2 above)
Comment (LindseySaari) [reply lib/config_helper.rb:10]: I would propose this and related changes for the mailer preview functionality in staging be broken out into a separate PR please

Comment (zurbergram) [reply lib/config_helper.rb:10]: https://github.com/department-of-veterans-affairs/vets-api/pull/5557

**Comment by LindseySaari** [comment]
Looks like some associated specs are failing

File: db/schema.rb (hunk 1 above)
Comment (LindseySaari) [reply db/schema.rb:325]: It appears that your branch doesn't have this change: https://github.com/department-of-veterans-affairs/vets-api/commit/42b7769ffba861692eb5bdabfceb0b148c2bb52c#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3

Comment (zurbergram) [reply db/schema.rb]: i haven't done a `make migrate` in a while which is why was continuing to show up when doing a `make rebuild`

Comment (zurbergram) [reply db/schema.rb]: I removed this change but even after a `make migrate` > `make rebuild` the change still shows up

**Comment by zurbergram** [comment]
> Looks like some associated specs are failing

@LindseySaari I fixed related specs that were failing but other specs are failing

**Review by delli-gatti-michael — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**